app/src/main/java/com/jshvarts/healthreads/data/Api.kt

```diff
@@ -4,11 +4,21 @@ import com.jshvarts.healthreads.BuildConfig
 import com.jshvarts.healthreads.domain.Book
 import com.jshvarts.healthreads.domain.WrappedBookList
 import retrofit2.http.GET
+import retrofit2.http.Headers
+
+const val CACHE_CONTROL_HEADER = "Cache-Control"
+const val CACHE_CONTROL_NO_CACHE = "no-cache"
 
 private const val API_KEY = BuildConfig.NYT_API_KEY
+private const val FETCH_BOOKS_URL = "lists/current/health.json?api-key=$API_KEY"
 
 interface Api {
-  @GET(value = "lists/current/health.json?api-key=$API_KEY")
+  @GET(value = FETCH_BOOKS_URL)
   @WrappedBookList
   suspend fun fetchBooks(): List<Book>
+
+  @GET(value = FETCH_BOOKS_URL)
+  @WrappedBookList
+  @Headers("$CACHE_CONTROL_HEADER: $CACHE_CONTROL_NO_CACHE")
+  suspend fun fetchBooksForceRefresh(): List<Book>
```

Comment:
cache header can be controlled dynamically with `@Header("Cache-Control") String cacheControl` parameter but I decided against exposing this API to consumers which could potentially lead to misuse. Instead, I decided to add a separate api function with a name clearly describing its purpose. 

If `@Headers("Cache-Control: no-cache"` is not present, the api call will use cache by default.